crypto/dilithium/pqcrystals-dilithium_dilithium3_ref/symmetric-shake.c

```diff
@@ -0,0 +1,30 @@
+#include <stdint.h>
+#include "params.h"
+#include "symmetric.h"
+#include "fips202.h"
+
+#ifndef DILITHIUM_USE_AES
```

Comment:
Why this change from upstream?

Reply:
Removed it in https://github.com/awslabs/aws-lc/commit/65221e3f3983d4e00e635bb4f61fe471e00cf611. If the flag `DILITHIUM_USE_AES` was turned on this file would fail to compile, as it uses SHAKE instead of AES. I added this `#ifndef` to fix this issue. However, if we are removing the support for AES modes then the issue will not arise.